api/client/options.proto

```diff
@@ -14,6 +14,19 @@ message RequestOptions {
   google.protobuf.UInt32Value request_body_size = 3 [(validate.rules).uint32 = {lte: 4194304}];
 }
 
+// We wrap all values so we can have a unified way of option handling with respect to
+// defaults, merging, etc. As there's no stock concept for enumerations, we manually
+// define custom wrappers for them. These used to be strings, which did provide the
+// wrapped type.
+message AddressFamily {
+  enum AddressFamilyOptions {
+    v4 = 0;
```

Comment:
Please use all caps for enum constants, e.g. V4. Also, the default (with value 0) should be either UNSET, DEFAULT or something you know is always going to be a sensible default and never go away, e.g. AUTO in this case.

Reply:
Fixed